File: year2020/month09/day0914/ms_0914.cpp
#include <iostream>

using namespace std;

void printRec(int n);

int main() {
	int n;

	cin >> n;

	if (1 <= n && n <= 100)
		printRec(n);
	
	return 0;
}

void printRec(int n) {
	for (int i = 1; i <= n; i++) {
		for (int j = 1; j <= n; j++) {
			cout << i * j;

			if (j != n)
				cout << ' ';
		}

		cout << endl;
	}
}

File: year2020/month09/day0914/ms_0914.cs
using System;

namespace algorithm_56
{
    class ms_0914
    {
        static void printRec(int n) 
        {
            for(int i = 1; i <= n; i++) 
            {
                for(int j = 1; j <= n; j++) 
                {
                    Console.Write(i*j);

                    if(j != n)
                        Console.Write(' ');
                }

                Console.WriteLine();
            }
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(1 <= n && n <= 100)
                printRec(n);
        }
    }
}
